packages/common/src/services/local-storage/LocalStorage.ts

```diff
@@ -20,6 +20,11 @@ type LocalStorageConfig = {
   localStorage: LocalStorageType
 }
 
+export type CachedDiscoveryProviderType = {
```

Comment:
It sort of makes me sad that this lives in the general LocalStorage.ts file, but I see the pattern was already there 😢

Reply:
Yeah agree